src/diff.test.ts

```diff
@@ -49,7 +49,7 @@ test('empty section', () => {
         'bundles',
       )} got bigger`,
       isEmpty: true,
-      children: renderBiggerTable({ assets: [] }),
```

Comment:
**nit** Is this a formatting / prettier thing?

Reply:
**suggestion** Could you add a test case for this new section?

Given how often we touch this repo and how relatively few people touch it, the easier we can make it for folks to jump in the better. And these test cases can help by capturing the various features we support.

Reply:
**suggestion** Could you add a test case for this new section?

Given how often we touch this repo and how relatively few people touch it, the easier we can make it for folks to jump in the better. And these test cases can help by capturing the various features we support.

Reply:
no excuse on not adding test cases, btw I added a github action to prettier check, build, package and test however in our [gitignore](https://github.com/launchdarkly-labs/webpack-bundle-diff-action/blob/main/.gitignore#L6) file we exclude `**-webpack-**.json` which makes the newly added github action steps on this repo render this error. 

<img width="974" alt="Screen Shot 2022-05-11 at 1 46 12 AM" src="https://user-images.githubusercontent.com/13124163/167785729-ea1d981f-5af2-48b6-a81e-ac690722a111.png">

Reply:
Ugh, I must have forgotten to check in those test reports. Let me see if I can dig them up from my old laptop.